go/storage/mkvs/urkel/fuzz/tree.go

```diff
@@ -9,29 +9,58 @@ import (
 	"encoding/json"
 	"fmt"
 	"sort"
+	"io/ioutil"
 
+	"github.com/oasislabs/oasis-core/go/common"
 	commonFuzz "github.com/oasislabs/oasis-core/go/common/fuzz"
 	mkvs "github.com/oasislabs/oasis-core/go/storage/mkvs/urkel"
+	mkvsNode "github.com/oasislabs/oasis-core/go/storage/mkvs/urkel/node"
 	mkvsTests "github.com/oasislabs/oasis-core/go/storage/mkvs/urkel/tests"
 )
 
 var treeFuzzer *commonFuzz.InterfaceFuzzer
 
-// TreeFuzz is a wrapper around a mkvs.KeyValueTree for fuzzing purposes.
+// TreeFuzz is a wrapper around a mkvs.Tree for fuzzing purposes.
 type TreeFuzz struct {
-	inner     mkvs.KeyValueTree
+	inner     mkvs.Tree
+	remote    mkvs.Tree
```

Comment:
:information_source: this starts out nil, and there's a bunch of `tree == nil` checks in the ops. then later `commitRemote` sets it